e2e/e2e_test.go

```diff
@@ -99,6 +99,31 @@ func TestE2EBasic(t *testing.T) {
 	_ = cleanup()
 }
 
+// TestE2ENoTasks runs the CTS binary in daemon mode with a configuration with no
+// tasks
+func TestE2ENoTasks(t *testing.T) {
```

Comment:
This is some super thorough testing with all the e2e test updates. I noticed that TestE2E_CreateTaskCommand and TestE2E_CreateDeleteCreateTrigger were also updated. For me, the no-task case is a detail-related to the command tests but not one of its core test case. But let me know your thought process.

One option could be to keep the command-related tests as they were and update the TestE2ENoTasks to include command testing. I'm wondering if it could be as simple as having TestE2ENoTasks call runCreateDeleteCreateTrigger()? I'm assuming it covers the test case in TestE2E_CreateTaskCommand but that might be incorrect.

We may have different thoughts / preferences. Let me know what you think!

Reply:
yeah, I can do that, then remove that level of testing from the commands